packages/node_modules/@webex/plugin-meetings/src/meeting/index.js

```diff
@@ -1616,9 +1616,35 @@ export default class Meeting extends StatelessWebexPlugin {
         VIDEO_TRACK: '1',
         SHARE_TRACK: '2'
       };
-      const trackMediaID = event.transceiver.mid;
       let eventType = null;
       const mediaTrack = event.track;
+      let trackMediaID = null;
+
+
+      // **In case of safari some time the transceiver is not present for specific os version
```

Comment:
can you remove this `**` please? 🙈

Reply:
we need this  to track this is a hack for safari